e2e/cypress/support/compliance_export_commands.js

```diff
@@ -0,0 +1,176 @@
+        thumbnailSize: {width: 400, height: 400},
+    };
+    cy.get('#fileUploadInput').attachFile(file.filename);
+
+    // # Wait until the image is uploaded
+    cy.waitUntil(() => cy.get('#postCreateFooter').then((el) => {
+        return el.find('.post-image.normal').length > 0;
+    }), {
+        timeout: TIMEOUTS.FIVE_MIN,
+        interval: TIMEOUTS.ONE_SEC,
+        errorMsg: 'Unable to upload attachment in time',
+    });
+
+    cy.postMessage(`file uploaded-${file.filename}`);
+});
+
+Cypress.Commands.add('exportCompliance', () => {
+    // # Click the export job button
+    cy.contains('button', 'Run Compliance Export Job Now').click();
+
+    // # Small wait to ensure new row is add
+    cy.wait(TIMEOUTS.THREE_SEC);
+
+    // # Get the first row
+    cy.get('.job-table__table').
+        find('tbody > tr').
+        eq(0).
+        as('firstRow');
+
+    // # Get the first table header
+    cy.get('.job-table__table').find('thead > tr').as('firstheader');
+
+    // # Wait until export is finished
+    cy.waitUntil(() => {
+        return cy.get('@firstRow').find('td:eq(1)').then((el) => {
+            return el[0].innerText.trim() === 'Success';
+        });
+    },
+    {
+        timeout: TIMEOUTS.FIVE_MIN,
+        interval: TIMEOUTS.ONE_SEC,
+        errorMsg: 'Compliance export did not finish in time',
+    });
+});
+
+Cypress.Commands.add('downloadAttachmentAndVerifyItsProperties', (fileURL = {}) => {
+    cy.request(fileURL).then((response) => {
+        // * Verify the download
+        expect(response.status).to.equal(200);
+
+        // * Confirm it's a zip file
+        expect(response.headers['content-type']).to.equal('application/zip');
+    });
+});
+
+Cypress.Commands.add('deleteLastPost', () => {
+    cy.apiGetTeamsForUser().then(({teams}) => {
+        const team = teams[0];
+        cy.visit(`/${team.name}/channels/town-square`);
+        cy.getLastPostId().then((lastPostId) => {
+        // # Click post dot menu in center.
+            cy.clickPostDotMenu(lastPostId);
+
+            // # Scan inside the post menu dropdown
+            cy.get(`#CENTER_dropdown_${lastPostId}`).should('exist').within(() => {
+                // # Click on the delete post button from the dropdown
+                cy.findByText('Delete').should('exist').click();
+            });
+        });
+        cy.get('.a11y__modal.modal-dialog').should('exist').and('be.visible').
+            within(() => {
+                // # Confirm click on the delete button for the post
+                cy.findByText('Delete').should('be.visible').click();
+            });
+    });
+});
+
+Cypress.Commands.add('gotoTeam', () => {
+    cy.apiGetTeamsForUser().then(({teams}) => {
+        const team = teams[0];
+        cy.visit(`/${team.name}/channels/town-square`);
+        cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible');
+    });
+});
+
+Cypress.Commands.add('postBOTMessage', (newTeam, newChannel, botId, botName, message) => {
```

Comment:
> Refer to existing [cy.postBotMessage](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/task_commands.js#L88) and see if it can be of use.

I have moved this method to local as it is taking care of token as well and not being used in other specs.